vignettes/Treatment.Rmd

```diff
@@ -15,50 +15,126 @@ knitr::opts_chunk$set(
 ```
 
 ```{r setup}
-suppressPackageStartupMessages(library(ggplot2))
 library(malariasimulation)
-library(malariaEquilibrium)
 ```
 
-Set up some baseline parameters:
+# Introduction
+
+This vignette describes how to implement clinical drug treatments in malariasimulation.
+
+The malariasimulation package contains in-built parameters sets for three anti-malarial drugs:
+
+1.  `AL_params`: artemether-lumefantrine (AL)
+2.  `DHA_PQP_params`: dihydroartemisinin and piperaquine (DHA-PQP)
+3.  `SP_AQ_params`: sulfadoxine-pyrimethamine and amodiaquine (SP-AQ)
+
+While all these drugs can be used to treat malaria, DHA-PQP and SP-AQ remain in the body for some time following treatment, making them good candidates for chemoprevention in mass drug administrations (see [Mass Drug Administration](https://mrc-ide.github.io/malariasimulation/articles/MDA.html). Any of these drugs can be included in the parameter list using the `set_drugs()` function (see [drugs_parameters.R](https://github.com/mrc-ide/malariasimulation/blob/master/R/drug_parameters.R) for full parameter details).
+
+Each drug parameter set is a vector of length four, that represent the drug efficacy, the infectiousness following treatment relative to an untreated infection, and parameters that determine the protection against reinfection ($P$): shape ($w$) and scale ($\lambda$). The decay of protection against infection follows a weibull distribution as follows:
```

Comment:
length four, with parameters that represent ...

Reply:
Done - thanks!